docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md

```diff
@@ -84,6 +80,23 @@ props.setProperty(ReadWriteSplittingPlugin.READER_HOST_SELECTOR_STRATEGY.name, "
+| `random`            | This strategy does not have configuration parameters. | The random strategy is the default connection strategy. When switching to a reader connection, the reader instance will be chosen randomly from the available database instances.                                                                                                                                                                                                                                                                            | N/A           |
+| `leastConnections`  | This strategy does not have configuration parameters. | The least connections strategy will select reader instances based on which database instance has the least number of currently active connections. Note that this strategy is only available when internal connection pools are enabled - if you set the connection property without enabling internal pools, an exception will be thrown.                                                                                                                   | N/A           |
+| `roundRobin`        | See the following rows for configuration parameters.  | The round robin strategy will select a reader instance by taking turns with all available database instances in a cycle. A slight addition to the round robin strategy is the weighted round robin strategy, where more connections will be passed to reader instances based on user specified connection properties.                                                                                                                                        | N/A           |
+|                     | `roundRobinHostWeights`                               | This parameter value must be a `string` type comma separated list of database host-weight pairs in the format `<host>:<weight>`. The host represents the database instance name, and the weight represents how many connections should be directed to the host in one cycle through all available hosts. For example, the value `instance-1:1,instance-2:4` means that for every connection to `instance-1`, there will be four connections to `instance-2`. | `null`        |
+|                     | `roundRobinDefaultWeight`                             | This parameter value must be a numeric value in the form of a `string`. This parameter represents the default weight for any hosts that have not been configured with the `roundRobinHostWeights` parameter. For example, if a connection were already established and host weights were set with `roundRobinHostWeights` but a new reader node was added to the database, the new reader node would use the default weight.                                 | `1`           |
+
+To indicate which connection strategy to use, the `readerHostSelectorStrategy` configuration parameter can be set. The following is an example of enabling the least connections strategy:
```

Comment:
Minor Nit: To me, it makes more sense to describe how to set `readerHostSelectorStrategy` before what options there are.